### DISCUSSION TITLE
Published about WebCrumbs? Share here!

### DISCUSSION THREAD

**Discussion by webcrumbs-community** [webcrumbs-community/webcrumbs#109] [Show and tell]
Hey there, WebCrumbs enthusiasts!

We're on the lookout for your stories, tutorials, hacks, and every slice of knowledge you've baked that mentions WebCrumbs. If you've penned an article, crafted a tutorial, or want to share how WebCrumbs will simplify our web development journey, we want to hear about it!

Sharing not only helps others in the community but also gives your work visibility among peers who share your passion. Plus, it's a great way to get feedback and engage in discussions that could spark new ideas or lead to potential collaborations.

Here’s how to share:

1. Write a piece on your preferred platform (Medium, Dev.to, your personal blog, etc.).
2. Include a link back to WebCrumbs GitHub Repository https://github.com/webcrumbs-community/webcrumbs
3. You can also include a link to our website with an UTM, such as https://webcrumbs.org?utm_source=REPLACE_WITH_PLATFORM&utm_campaign=REPLACE_WITH_USERNAME
4. Paste the link to your article in the comments below with a brief description.
5. Feel free to tag us or use the hashtag #WebCrumbs and our twitter @BuildWebCrumbs when you share your article on social media.

Whether you're a seasoned developer or just getting started, your insights are valuable. Let's create a repository of knowledge that helps the entire community grow. Plus, selected articles will get a shoutout in our community channels!

Can't wait to read about your experiences, learn from your expertise, and share in your development adventures!

Let the sharing begin!

P.S. If you haven't yet contributed but are interested, check out our GitHub repo [webcrumbs-community/webcrumbs](https://github.com/webcrumbs-community/webcrumbs) and become part of our growing global community!

We’re thrilled to see how you're weaving WebCrumbs into your web dev narratives!

**Comment by AJBrownson** [comment]
@webcrumbs-community Okay 
We could certainly share our articles here

**Reply by webcrumbs-community** [reply]
Thanks for bringing this up, @AJBrownson!

**Comment by webcrumbs-community** [comment]
This one here may help for those that like posting on dev.to. Cheers! 
https://dev.to/opensourcee/trying-out-some-cool-devto-tricks-i-just-found-out-about-1572

**Comment by webcrumbs-community** [comment]
Fresh from the press 🙏
https://dev.to/opensourcee/what-is-webcrumbs-m58

**Comment by webcrumbs-community** [comment]
Absolutely thrilled to witness the enthusiasm from our friends in Japan championing WebCrumbs! Check out their spotlight on us: http://kachibito.net/useful-resource/webcrumbs. Let's keep this collaborative spirit soaring and push the boundaries of web development further together! 🚀🌐 #CommunityPower #WebCrumbsGlobal

**Reply by webcrumbs-community** [reply]
We were also featured at https://b.hatena.ne.jp/ ☺️ (_Hatena s an internet services company in [Japan](https://en.wikipedia.org/wiki/Japan) that operates the most popular [social bookmarking](https://en.wikipedia.org/wiki/Social_bookmarking) service in Japan,[[1]](https://en.wikipedia.org/wiki/Hatena_(company)#cite_note-1) Hatena Bookmark._)

**Comment by webcrumbs-community** [comment]
Posted this, mentioning WebCrumbs and @aizhuxue007:
https://dev.to/opensourcee/state-management-in-react-finding-the-sweet-spot-5efh

**Comment by webcrumbs-community** [comment]
Fresh from the press 🖋️
https://dev.to/opensourcee/the-story-behind-webcrumbs-4k6n
